### PR TITLE
Changed README just a little

### PR DESCRIPTION
**Pull request by jgx0**

### CODE REVIEW

File: log.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+2020/10/19 12:26:16 Micro started
+2020/10/19 12:26:16 monokai-dark is not a plugin
```
Comment (vainiovano) [inline log.txt]: What is this?

**Review by TheRyeGuyWhoWillNowDie — DISMISSED** [review]
kind of weird but ok

File: log.txt (hunk 1 above)
Comment (jgx0) [reply log.txt]: sorry its my git log you can delete that

**Review by monban — APPROVED** [review]
Weird flex but okay.

**Pull request merged**